command/aarelayer/service/aa_relayer_service.go

```diff
@@ -9,9 +9,12 @@ import (
 	"time"
 
 	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
 	"github.com/umbracle/ethgo"
 )
 
+const receiptSuccess = 1
```

Comment:
Use `types.ReceiptSuccess` instead.
```suggestion
```

Reply:
I know, but this will be moved to separate repository, so I am trying to depend on edge fw as less as I can

Reply:
Ok, though you are already relying on types package, so it makes sense to start decoupling it.